src/core/ui/applyPatch.ts

```diff
@@ -0,0 +1,147 @@
+import { GraphPatch } from '../types';
+
+export type Point = { x: number; y: number };
+
+export type UiNodeShape = 'Circle' | 'Square';
+
+export interface UiNodePatchProperties {
+	backgroundColor: string;
+	borderColor: string;
+	highlightBorderColor: string;
+	highlight: boolean;
+	label: string;
+	shape: UiNodeShape;
+	nodeTemplate: string;
+	symbolId: string;
+	symbol: UiNodeSymbol;
+	symbolWidth: number;
+	symbolHeight: number;
+	symbolGeometry: string;
+}
+
+export interface UiEdge {
+	edgeId: string;
+	fromNode: string;
+	fromConnector?: string;
+	toNode: string;
+	toConnector?: string;
+}
+
+export interface UiEdgePatchProperties {
+	color: string;
+	highlightColor: string;
+	highlight: boolean;
+	thickness: string;
+}
+
+export interface UiConnectorPatchProperties {
+	name: string;
+	color: string;
+	connectorRelativePosition: Point | 'Left' | 'Right' | 'Top' | 'Bottom';
+	/** The angle that the connector arm has to the node at default rotation */
+	connectorDirection: number;
```

Comment:
```suggestion
	connectorDirection: number; // The angle that the connector arm has to the node at default rotation
```

Minor, but find it more trivial to match code with comment this way

Reply:
Then we don't se this description when you hover the property in another file :(